### DISCUSSION TITLE
Task Template - Run a task when a new commit is added to the repository

### DISCUSSION THREAD

**Discussion by pierrickj** [semaphoreui/semaphore#2183] [General]
Hello,

Just starting playing with semaphore.

My first tasks runs well but i'm stuck with the option :

Run a task when a new commit is added to the repository

Should i have additional configuration to do in addition to tick checkbox, select repos and check interval to make the task running once  there's new commit ?

![image](https://github.com/semaphoreui/semaphore/assets/114187002/bc669550-40c3-40fc-a921-351efe4ddb04)

it seems not working on my side.

Running 2.10.12-beta-59ef3f8-171978542

Any helps appreciated :)

**Comment by lwandrebeck** [comment]
Have a look at https://github.com/semaphoreui/semaphore/issues/2267
HTH
